docker/Dockerfile

```diff
@@ -90,22 +90,23 @@ COPY nautobot /source/nautobot
 
 # Build the whl for use in the final container and install for the dev container
 RUN poetry build && \
-    poetry install --no-ansi --extras mysql
+    poetry install --no-ansi --extras all
 
 ################################ Stage: cleaninstall
 # Build an image with required dependencies to pull the installation from
 FROM base as cleaninstall
 
 # hadolint ignore=DL3008
-RUN apt-get update && apt-get install --no-install-recommends -y build-essential libssl-dev
+RUN apt-get update && apt-get install --no-install-recommends -y build-essential libssl-dev libxmlsec1-dev libxmlsec1-openssl pkg-config default-libmysqlclient-dev libldap-dev libsasl2-dev
 
 # pyuwsgi doesn't support ssl so we build it from source
 # https://github.com/nautobot/nautobot/issues/193
 RUN pip install --no-cache-dir --no-binary=pyuwsgi pyuwsgi
 
 COPY --from=dev /source/dist/*.whl /tmp
 
-RUN pip install --no-cache-dir /tmp/nautobot*.whl
+# hadolint ignore=DL3013,SC2102
+RUN for whl in /tmp/nautobot*.whl; do pip install --no-cache-dir ${whl}[all]; done
```

Comment:
Just double-checking - we don't need to escape the `[]` characters here or anything like that, right?

Reply:
correct